front_end/content.py

```diff
@@ -1014,10 +1121,17 @@ def copy_assignment(self, course_id, assignment_id, new_course_id):
 
     def update_user(self, user_id, user_dict):
         sql = '''UPDATE users
-                 SET user_json = ?
+                 SET name = ?, given_name = ?, family_name = ?, picture = ?, locale = ?
                  WHERE user_id = ?'''
 
-        self.cursor.execute(sql, (json.dumps(user_dict), user_id,))
+        self.cursor.execute(sql, (user_dict["name"], user_dict["given_name"], user_dict["family_name"],
+        user_dict["picture"], user_dict["locale"], user_id,))
+
+    def update_user_theme(self, user_id, theme):
```

Comment:
Let's make this more generic than just the theme. Maybe update_user_setting() or update_user_settings() so that in the future when we want to support additional settings, we can use the same function.